src/language/enum.md

```diff
@@ -152,6 +156,13 @@ use the `.name` property:
 print(Color.blue.name); // 'blue'
 ```
 
+You can invoke a method on an enhanced enum just like you would on a class:
```

Comment:
I think this is necessary because while we do show `index` and `name` - both of those are "special" - they're not a method that the user introduced on the enum?